tests/php/sync/test_class.jetpack-sync-callables.php

```diff
@@ -55,7 +55,7 @@ public function test_sync_callable_whitelist() {
 		if ( function_exists( 'register_block_type' ) ) {
```

Comment:
Could we remove the check here now?

Reply:
I'll see if that works with WP<5.0 without requiring modifications to the `Jetpack_Gutenberg` class -- otherwise I'll defer to a follow-up PR if that's okay.

Reply:
Works: a4e71b18a